generic_throttle/README.md

```diff
@@ -0,0 +1,20 @@
+# generic_throttle
+
+generic_throttle is package offering a Python implementation of a throttle for topics.
+
+Class GenericThrottle runs a ROS node which subscribes to a list of topics and
```

Comment:
~~Am I missing something: there is no node that actually uses the GenericThrottle python module...
Are you going to add one? Where/How are you currently using this package/node?~~

Found the ```rospy.init_node```.
But still there is now ```__main__```/executable, right?

I'd rather have an executable python ```generic_throttle_node.py``` which makes use of ```generic_throttle.GenericThrottle``` class but does all the "node initialization" and spinning...